doc/source/release_notes.rst

```diff
@@ -55,6 +41,121 @@ API Changes
       :func:`ophyd.setup_ophyd`, which is still available as a top-level
       import.
 
+* For complex devices, the same child device may be used in both ``read_attrs``
```

Comment:
This is missing "When recursing through complex devices ``read`` in no longer called as part of ``read_configuration``"